docs/userguide/insights.md

```diff
@@ -142,4 +142,16 @@ insight.plot("bar")
 
 As expected, the result from [`LimeTabular`](shap.explainers.other.LimeTabular) are very
 similar to the results from the SHAP [`KernelExplainer`](shap.KernelExplainer) because
-both methods involve linear local approximations.
\ No newline at end of file
+both methods involve linear local approximations.
+
+### Multi-Output Surrogate Models
+In case you are using a surrogate model with multiple outputs, e.g. while using 
+a [`CompositeSurrogate`](baybe.surrogates.composite.CompositeSurrogate) 
+for a [`ParetoObjective`](baybe.objectives.pareto.ParetoObjective), 
+there are multiple models to be analyzed.
```

Comment:
The "while" sounds a bit weird?
```suggestion
In case you are using a surrogate model with multiple outputs,
e.g. a [`CompositeSurrogate`](baybe.surrogates.composite.CompositeSurrogate) 
for a [`ParetoObjective`](baybe.objectives.pareto.ParetoObjective), 
there are multiple models to be analyzed.
```